plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -187,6 +217,41 @@ def setup(
         )
         self.sweep_dir = config.hydra.sweep.dir
 
+    def _get_directions(self) -> List[str]:
+        if isinstance(self.direction, MutableSequence):
+            return [d.name if isinstance(d, Direction) else d for d in self.direction]
+        elif isinstance(self.direction, str):
+            return [self.direction]
+        return [self.direction.name]
+
+    def _configure_trials(
+        self,
+        trials: List[Trial],
+        search_space: Dict[str, BaseDistribution],
```

Comment:
Same remark as before about renaming `search_space`.

Reply:
Changed it to search_space_distributions